content/en/opentelemetry/compatibility.md

```diff
@@ -0,0 +1,87 @@
+| Live Processes                         |                                    | {{< X >}}                          | {{< X >}}                           |                                                | {{< X >}}                          |
+| Mobile SDKs                            |                                    |                                    | {{< X >}}                           |                                                | {{< X >}}                          |
+| Universal Service Monitoring (USM)     |                                    |                                    | {{< X >}}                           |                                                | {{< X >}}                          |
+| Cloud SIEM                             | {{< X >}}                          | {{< X >}}                          | {{< X >}}                           | {{< X >}}                                      | {{< X >}}                          |
+| Correlated Traces, Metrics, Logs       | {{< X >}}                          | {{< X >}}                          | {{< X >}}                           | {{< X >}}                                      | {{< X >}}                          |
+| Live Container Monitoring/K8s Explorer |                                    | {{< X >}}                          | {{< X >}}                           | {{< X >}}                                      | {{< X >}}                          |
+| Network Performance Monitoring (NPM)   |                                    | {{< X >}}                          | {{< X >}}                           | {{< X >}}                                      | {{< X >}}                          |
+| Real User Monitoring (RUM)             | {{< X >}}                          | {{< X >}}                          | {{< X >}}                           | {{< X >}}                                      | {{< X >}}                          |
+| Runtime Metrics                        | {{< X >}}<br>(Java, .NET, Go only) | {{< X >}}<br>(Java, .NET, Go only) | {{< X >}}                           | {{< X >}}                                      | {{< X >}}<br>(Java, .NET, Go only) |
+| Source code integration                | {{< X >}}                          | {{< X >}}                          | {{< X >}}                           |                                                | {{< X >}}                          |
+
+## More details
+
+### Runtime Metrics
+
+Setups using the OpenTelemetry SDK follow the [OpenTelemetry Runtime Metrics][1] specification.
+
+### Real User Monitoring (RUM)
+
+To enable full RUM functionality, you need to [inject supported headers][2] to correlate RUM and traces.
+
+### Network Performance Monitoring (NPM)
+
+For **OTel to Datadog (OTLP)** and **OTel with embedded Datadog Collector (Preview)** setups, span-level or endpoint-level monitoring is **not** supported.
```

Comment:
span-level or endpoint-level monitoring is actually not supported for any NPM including full datadog. we can either omit or clarify

Reply:
Gotcha, I just clarified that this is always the case.